examples/start_building/v2/go/yourFirstTransaction.go

```diff
@@ -3,60 +3,18 @@ package main
 import (
 	"context"
 	json "encoding/json"
-	"errors"
+	// "errors"
 	"fmt"
-	"strings"
-
+	// "strings"
+	"github.com/algorand/go-algorand-sdk/future"
 	"github.com/algorand/go-algorand-sdk/client/v2/algod"
-	"github.com/algorand/go-algorand-sdk/client/v2/common/models"
+	// "github.com/algorand/go-algorand-sdk/client/v2/common/models"
```

Comment:
can you just remove these commented out lines please?

Reply:
done